### PR TITLE
Added new fingerprint for Skoda Superb Mk3

### PR DESCRIPTION
**Pull request by lskov**
Added a new fingerprint for Skoda under Volkswagen MQB platform and tested on my own vehicle .

### CODE REVIEW

**Comment by pd0wm** [comment]
What's your dongle id?

**Comment by lskov** [comment]
7291d51974eb2d1d

/Lasse Skov

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -552,6 +554,7 @@ class CAR:
       b'\xf1\x875Q0909143M \xf1\x892041\xf1\x820522UZ070303',
       b'\xf1\x875Q0910143B \xf1\x892201\xf1\x82\00563UZ060700',
       b'\xf1\x875Q0909143K \xf1\x892033\xf1\x820514UZ070203',
+      b'\xf1\x875Q0910143B \xf1\x892201\xf1\x82\00563UZ060600',
```
Comment (pd0wm) [inline selfdrive/car/volkswagen/values.py:557]:
```suggestion
      b'\xf1\x875Q0910143B \xf1\x892201\xf1\x82\x0563UZ060600',
```

Comment (pd0wm) [reply selfdrive/car/volkswagen/values.py:557]: Use hex notation instead of octal

**Pull request merged**